### PR TITLE
dispatch the broken link to a branch

### PR DESCRIPTION
**Pull request by LinshanYu**
dispatch the broken link to a branch

---
This change is [Reviewable](https://reviewable.io/reviews/kubernetes/cloud-provider-openstack/990)

### CODE REVIEW

**Comment by jichenjc** [comment]
um.. why point to a branch ? didn't get the point? if master is broken, why not make it workable?

File: docs/using-cinder-csi-plugin.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Check [kubernetes CSI Docs](https://kubernetes-csi.github.io/docs/) for flag det
 
 ### Deploy
 
-If you already created the `cloud-config` secret used by the [cloud-controller-manager](https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/using-controller-manager-with-kubeadm.md#steps), remove the file ```manifests/cinder-csi-plugin/csi-secret-cinderplugin.yaml``` and then jump directly to the `kubectl apply ...` command.
+If you already created the `cloud-config` secret used by the [cloud-controller-manager](https://github.com/kubernetes/cloud-provider-openstack/blob/release-1.9/docs/using-controller-manager-with-kubeadm.md#steps), remove the file ```manifests/cinder-csi-plugin/csi-secret-cinderplugin.yaml``` and then jump directly to the `kubectl apply ...` command.
```
Comment (ramineni) [inline docs/using-cinder-csi-plugin.md]: correct link would be , https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/using-openstack-cloud-controller-manager.md

**Comment by LinshanYu** [comment]
> um.. why point to a branch ? didn't get the point? if master is broken, why not make it workable?

because master branch changes frequently, causing some links to fail。

File: docs/using-cinder-csi-plugin.md (hunk 1 above)
Comment (LinshanYu) [reply docs/using-cinder-csi-plugin.md]: ah...I think you may be right

Comment (LinshanYu) [reply docs/using-cinder-csi-plugin.md]: > correct link would be , https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/using-openstack-cloud-controller-manager.md
I think you may be right

**Comment by lingxiankong** [comment]
Please modify the PR description according to https://raw.githubusercontent.com/kubernetes/cloud-provider-openstack/master/.github/PULL_REQUEST_TEMPLATE.md

/hold

**Pull request closed without merging**